src/main/java/dk/cngroup/lentils/repository/HintRepository.java

```diff
@@ -16,7 +16,8 @@
     @Query(value = "SELECT DISTINCT h.* FROM hint h " +
             "LEFT JOIN hint_taken ht ON ht.hint_hint_id = h.hint_id " +
             "WHERE h.hint_id NOT IN (SELECT ht2.hint_hint_id FROM hint_taken ht2 " +
-            "WHERE ht2.team_team_id = ?1) AND h.cypher_id = ?2",
+            "WHERE ht2.team_team_id = ?1) AND h.cypher_id = ?2 " +
+            "ORDER BY h.VALUE ASC",
```

Comment:
Prosim `h.VALUE` zmenit na `h.value` (Sloupce pisem v tech queries malymi pismey, tak at je to jednotne. Zmena ve druhem souboru uz je v poradku)

Reply:
jasne, díky :)